src/test/java/careconnect/logic/commands/CommandTestUtil.java

```diff
@@ -47,16 +54,32 @@ public class CommandTestUtil {
     public static final String ADDRESS_DESC_BOB = " " + PREFIX_ADDRESS + VALID_ADDRESS_BOB;
     public static final String TAG_DESC_FRIEND = " " + PREFIX_TAG + VALID_TAG_FRIEND;
     public static final String TAG_DESC_HUSBAND = " " + PREFIX_TAG + VALID_TAG_HUSBAND;
+    public static final String LOG_DESC_DATE = " " + PREFIX_DATE + "2020-12-31 12:00";
+    public static final String LOG_DESC_REMARK = " " + PREFIX_REMARK + "Meeting 1";
+    public static final String LOG_DELETE_INDEX = " " + PREFIX_LOG_INDEX + "1";
 
+    public static final String INVALID_INDEX_DESC = " " + "0"; // '0' not allowed for indexes
     public static final String INVALID_NAME_DESC = " " + PREFIX_NAME + "James&"; // '&' not allowed in names
     public static final String INVALID_PHONE_DESC = " " + PREFIX_PHONE + "911a"; // 'a' not allowed in phones
     public static final String INVALID_EMAIL_DESC = " " + PREFIX_EMAIL + "bob!yahoo"; // missing '@' symbol
     public static final String INVALID_ADDRESS_DESC = " " + PREFIX_ADDRESS; // empty string not allowed for addresses
     public static final String INVALID_TAG_DESC = " " + PREFIX_TAG + "hubby*"; // '*' not allowed in tags
+    public static final String INVALID_DATE_DESC = " " + PREFIX_DATE + "2020-12-31 12:00pm"; //
+    // invalid date format - 24 hours used
+    public static final String INVALID_REMARK_DESC = " " + PREFIX_REMARK + " "; // empty string
+    // not allowed for remarks
 
     public static final String PREAMBLE_WHITESPACE = "\t  \r  \n";
     public static final String PREAMBLE_NON_EMPTY = "NonEmptyPreamble";
 
+    public static final Date DATE = Date.from(
+            LocalDateTime.of(2020, 12, 31, 12, 0)
+                    .atZone(ZoneId.of("UTC")).toInstant());
+    public static final Date DATE2 = Date.from(
+            LocalDateTime.of(2021, 11, 10, 11, 0)
+                    .atZone(ZoneId.of("UTC")).toInstant());
```

Comment:
if we choose to change to singapore time, should change the timezone to "Asia/Singapore"